crates/telio-model/src/features.rs

```diff
@@ -487,7 +498,10 @@ mod tests {
             "pmtu_discovery": {
                 "response_wait_timeout_s": 20
             },
-            "multicast": true
+            "multicast": true,
+            "batching": {
```

Comment:
As we now have a feature builder, it would be nice to have a `enable_batching()` option, see src/ffi/defaults_builder

Reply:
Sure thing. Did that :)